L1Trigger/L1TMuonEndCap/src/TrackFinder.cc

```diff
@@ -244,7 +243,7 @@ void TrackFinder::process(
 
             } else if (h.Subsystem() == TriggerPrimitive::kRPC) {
               if (h.Sector_idx() != es)  continue;
-              if (h.BX()+5 != ibx)  continue;  // RPC hits should be supplied 5 BX later relative to CSC hits
+              if (h.BX()+6 != ibx)  continue;  // RPC hits should be supplied 6 BX later relative to CSC hits
```

Comment:
Was all our RPC emulation off by 1 BX before?

Reply:
No, this has nothing to do with the algorithm itself (as far as I know). This is just how Alex reads the hits from a text file into his firmware simulation.